app/models/reporting_pipeline/patient_blood_pressures_per_month.rb

```diff
@@ -0,0 +1,6 @@
+module ReportingPipeline
+  class PatientBloodPressuresPerMonth < Matview
+    self.table_name = "reporting_patient_blood_pressures_per_month"
+    belongs_to :patient
+  end
+end
```

Comment:
Can we avoid adding this model until we actually use it in code? I was even trying to avoid adding the `Visits` model, but we wanted to unit test that finely. This model's unit tests are already covered in the `States` model's unit tests.

Reply:
I was doing that initially, but then I discovered the cascade issue in testing and using this all locally...and I didn't want to add a manual refresh call for just _one_ of our views.

Reply:
We could also just call scenic's refresh with a given table name without creating a model. I was doing this before I saw the cascade option.